Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -1107,8 +1334,14 @@ def create_rule_object_command(client: Client, args: Dict) -> CommandResults:
         'FromAddress': from_address,
         'ToAddress': to_address
     }]
+    # create the body according to the version of the NSM
+    if VERSION == "V10x":
```

Comment:
let's create a const for 'V10x' and use it in all relevant places.

Reply:
In this case it will not save any code, is it still necessary?